setup.cfg

```diff
@@ -29,7 +29,7 @@ setup_requires =
     setuptools
 install_requires =
     click
-    docker>=2.0.2
+    docker>=3.0.0
```

Comment:
Maybe we should also set `<4` to prevent future such issues.

Reply:
By doing this, we will not see the new version major of docker(-py) and Grocker could be linked with an old and unmaintained version of docker(-py).